src/main/java/com/sobytylnik/hello/InMemoryProfileRepository.java

```diff
@@ -0,0 +1,65 @@
+package com.sobytylnik.hello;
+
+import java.util.Optional;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class InMemoryProfileRepository implements ProfileRepository {
+
+
+    private final ConcurrentHashMap<Long, Profile> ProfileMap;

```

Comment:
Follow naming convention for variables: https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-135099.html